### ISSUE TITLE
Reporting

### ISSUE THREAD

**Issue by bkamins** [pszufe/TransportationMetaheuristics.jl#2]
We probably should use loggers (`@info` & friends) for output with different reporting levels as now e.g. tabu search output is massively verbose.

**Comment by pszufe** [comment]
Yes - just for note https://docs.julialang.org/en/v1/stdlib/Logging/index.html
And the output needs to be reformatted.
